alttpo/receive.as

```diff
@@ -100,6 +100,7 @@ void receive() {
     players[index].deserialize(r, c);
     if (joined) {
       local.notify(players[index].name + " joined");
+      playersWindow.update();
```

Comment:
You'll want the corresponding "player left" case covered too. It's in `GameState.as` line 353.